open-api.yaml

```diff
@@ -0,0 +1,269 @@
+openapi: 3.0.3
+info:
+  title: ATLAS feeding_data
+  description: Send planned rations and get actual feeding reports into the system.
+  version: '0.0.2'
+servers:
+  - url: http://localhost:8080
+    description: Inferred Url
+security: 
+  - "bearerAuth": []
+paths:
+  /feedingreport:
+    get:
+      summary: feedingReport
+      parameters:
+        - name: start_date
+          in: query
+          description: Start date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: end_date
+          in: query
+          description: End date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: ration_ref
+          in: query
+          description: ration_ref represents identification for connecting the rations between systems
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RecipeDailyReport'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+  /ration:
+    post:
+      summary: importRation
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/RationRequest'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+        '201':
+          description: Created
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+        '409':
+          description: Validation failed
+components:
+  schemas:
+    URN:
+        type: "string"
+        pattern: "^urn:[a-z][a-z0-9_.\\-]*:[\\w-.]+"
+        description: "Uniform resource name of a field with the syntax `urn:<service_id>:<internal_id>`. <service_id> must be the id of a valid and active ATLAS service"
+        example: 'urn:animal_data:69d4a4ec-e8a4-4768-a299-14a7a9acd1b1'

```

Comment:
example is not very good since no service should allowed to take the generic animal_data id. Use de.fodjan.animal_data instead